oversweet-domain/src/main/java/com/depromeet/oversweet/domain/record/repository/FindDailyRecordsRepositoryImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.depromeet.oversweet.domain.record.repository;
+
+import com.depromeet.oversweet.domain.record.entity.RecordEntity;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import jakarta.persistence.EntityManager;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static com.depromeet.oversweet.domain.drink.entity.QDrinkEntity.drinkEntity;
+import static com.depromeet.oversweet.domain.franchise.entity.QFranchiseEntity.franchiseEntity;
+import static com.depromeet.oversweet.domain.member.entity.QMemberEntity.memberEntity;
+import static com.depromeet.oversweet.domain.record.entity.QRecordEntity.recordEntity;
+
+@Repository
+public class FindDailyRecordsRepositoryImpl implements FindDailyRecordsRepository {
+
+    private final JPAQueryFactory queryFactory;
+
+    public FindDailyRecordsRepositoryImpl(final EntityManager em) {
+        this.queryFactory = new JPAQueryFactory(em);
+    }
+
+    @Override
+    public List<RecordEntity> findDailyRecordsByLocalDatetime(final Long memberId, final LocalDateTime startDate, final LocalDateTime endDate) {
+
+        return queryFactory.selectFrom(recordEntity)
+                .join(recordEntity.member, memberEntity).fetchJoin() // xxToOne
+                .join(recordEntity.drink, drinkEntity).fetchJoin() // xxToOne
+                .join(recordEntity.drink.franchise, franchiseEntity).fetchJoin() // xxToOne
+                .where(memberEntity.id.eq(memberId))
+                .where(recordEntity.createdAt.between(startDate, endDate))
+                .fetch();
```

Comment:
Service Class의 코드를 보면
``` java
        // 유저의 하루 적정 섭취량 가져오기
        final MemberEntity findMember = findMemberRepository.findById(memberId);
```
이런식으로 하루 적정 섭취량을 가져오기 위해서 MemberEntity을 찾았는데

여기서 **또 다시 memberEntity와 join을 할 필요는 없어 보여요** 
```suggestion
        return queryFactory.selectFrom(recordEntity)
                .join(recordEntity.drink, drinkEntity).fetchJoin() // xxToOne
                .join(recordEntity.drink.franchise, franchiseEntity).fetchJoin() // xxToOne
                .where(memberEntity.id.eq(memberId))
                .where(recordEntity.createdAt.between(startDate, endDate))
                .fetch();
```

**MemberEntity와는 다시 Join 하지 않도록 이렇게만 가져가도 좋지 않을까요?**

Reply:
넵 감사합니다 `.where(memberEntity.id.eq(memberId))` 는 `.where(recordEntity.member.id.eq(memberId))`로 수정했습니당